inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.html

```diff
@@ -17,84 +17,112 @@
  - limitations under the License.
 -->
 <html xmlns:wicket="http://wicket.apache.org" xmlns="http://www.w3.org/1999/html" lang="">
-<head>
 <wicket:head>
-  <style type="text/css">
-.page-content {
- overflow: auto;
-}
+    <style type="text/css">
+        .page-content {
+            overflow: auto;
+        }
 
-.table-striped>tbody>tr:nth-child(2n+1)>td, .table-striped>tbody>tr:nth-child(2n+1)>th
- {
```

Comment:
why do we need to override the Bootstrap striped table color? https://getbootstrap.com/docs/4.5/content/tables/#striped-rows